.vscode/settings.json

```diff
@@ -16,5 +16,8 @@
     "source.fixAll": true
   },
   "cSpell.words": ["mantine"],
-  "vsicons.presets.nestjs": true
+  "vsicons.presets.nestjs": true,
+  "[markdown]": {
+    "editor.defaultFormatter": "DavidAnson.vscode-markdownlint"
+  }
```

Comment:
Please rollback this change.

Reply:
Resolved [here](https://github.com/novuhq/novu/pull/3590/commits/16ebbf0d257984db468396ca8f54b414ef322ea5)